utils/generateRandomNumber.js

```diff
@@ -0,0 +1,5 @@
+const { backupMessages } = require("../bin/backupMessages");
+
+const generateRandomNumber = Math.floor(Math.random() * (backupMessages.length - 1));
```

Comment:
Rather than importing backup messages, could the `generateRandomNumber` function take an `int` arg for the length of the data array